5-network/06-fetch-api/article.md

```diff
@@ -152,13 +152,13 @@ The `credentials` option specifies whether `fetch` should send cookies and HTTP-
 
 ## cache
 
-By default, `fetch` requests make use of standard HTTP-caching. That is, it honors `Expires`, `Cache-Control` headers, sends `If-Modified-Since`, and so on. Just like regular HTTP-requests do.
+By default, `fetch` requests make use of standard HTTP-caching. That is, it respects the `Expires` and `Cache-Control` headers, sends `If-Modified-Since` and so on. Just like regular HTTP-requests do.
 
 The `cache` options allows to ignore HTTP-cache or fine-tune its usage:
 
 - **`"default"`** -- `fetch` uses standard HTTP-cache rules and headers,
 - **`"no-store"`** -- totally ignore HTTP-cache, this mode becomes the default if we set a header `If-Modified-Since`, `If-None-Match`, `If-Unmodified-Since`, `If-Match`, or `If-Range`,
-- **`"reload"`** -- don't take the result from HTTP-cache (if any), but populate cache with the response (if response headers allow),
+- **`"reload"`** -- don't take the result from HTTP-cache (if any), but populate the cache with the response (if response headers are allowed),
```

Comment:
I meant: "if response headers allow that" =)

Reply:
> I meant: "if response headers allow that" =)

How about "if the response headers permit this action" ? :)

Reply:
A bit loo long, but that would be correct.

Reply:
Edited it. Decline if you prefer the old way